src/features/representativeOperatorContactDetails.feature

```diff
@@ -8,6 +8,7 @@ Feature: Representative Operator Contact Details
     Scenario: happy path without optional field
         Given I open the url "/cleansession"
         And I open the url "mid-and-east-antrim/contact-representative"
+        And I click on the element "repOpContactDetails.button"
```

Comment:
why is this addition needed?

Reply:
This is due to the 'accept all cookies' button that appears on clean sessions. the data was being input and when a continue button was being 'clicked', the test expected the page to be changed but the actual continue button wasnt being hit. This line clicks the 'accept cookies' button and then the page test starts

Reply:
ok maybe rename later